hack/jenkins/common.sh

```diff
@@ -70,17 +74,12 @@ if [ -e out/docker-machine-driver-hyperkit ]; then
   sudo chmod u+s out/docker-machine-driver-hyperkit || true
 fi
 
-MINIKUBE_WANTREPORTERRORPROMPT=False sudo ./out/minikube-${OS_ARCH} delete \
-|| MINIKUBE_WANTREPORTERRORPROMPT=False ./out/minikube-${OS_ARCH} delete \
-|| true
-sudo rm -rf $HOME/.minikube || true
-sudo rm -rf $HOME/.kube || true
-
 # See the default image
 ./out/minikube-${OS_ARCH} start -h | grep iso
 
 # see what driver we are using
 which docker-machine-driver-${VM_DRIVER} || true
+tree ~/.minikube || true
```

Comment:
How about "find ~/.minikube"? Not all systems have tree.

Reply:
I ended up putting it on the build slaves, find is probably simpler